contracts/Market.sol

```diff
@@ -57,9 +57,72 @@ contract Market is PausableAccessPreset {
 
   /**
    * @notice Emitted on setting of `_priorityRestrictedThreshold`.
+   * @param threshold The updated threshold for priority restricted supply.
    */
   event PriorityRestrictedThresholdSet(uint256 threshold);
 
+  /**
+   * @notice Emitted on setting of `_noriFeeWalletAddress`.
+   * @param updatedWalletAddress The updated address of the Nori fee wallet.
+   */
+  event NoriFeeWalletAddressUpdated(address updatedWalletAddress);
+
+  /**
+   * @notice Emitted on setting of `_noriFeePercentage`.
+   * @param updatedFeePercentage The updated fee percentage for Nori.
+   */
+  event NoriFeePercentageUpdated(uint256 updatedFeePercentage);
+
+  /**
+   * @notice Emitted on setting of `_currentSupplierAddress`.
+   * @param updatedSupplierAddress The updated supplier address.
+   */
+  event CurrentSupplierAddressChanged(address updatedSupplierAddress);
```

Comment:
I feel like this piece of state change actually should not involve emitting an event because this is likely to change many times in the course of fulfilling a single purchase and will add to the gas overhead for the buyer while not being particularly useful to us